pages/reservations.vue

```
@@ -201,12 +201,17 @@ const { locale } = useI18n()
-// Sites with experiences book per-experience on /experiences/[slug]; this generic
-// table-reservation form doesn't apply and is no longer linked from nav.
-if (hasExperiences.value) {
-  await navigateTo('/experiences', { replace: true, redirectCode: 302 })
-}
+const isExperienceSite = computed(() => (site as { vertical?: string | null } | null)?.vertical === 'experience')
+
+// Pure experience-vertical sites book per-experience on /experiences/[slug]; this
+// generic table-reservation form doesn't apply there. Restaurants that also have
+// experiences attached (e.g. Ember & Slice, Kikuzuki) still need this page for
+// regular table bookings alongside their experience bookings.
+watch([isExperienceSite, hasExperiences], ([isExp, hasExp]) => {
```

ESLint

[error] 'hasExp' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)